mage-os/mageos-magento2
# Update unit test for autocomplete suggests fix

Description (*)

Since 2.4.7 we noticed search autocomplete would suggest search terms with no result.

here are some more cases of bad search suggestions with sample data package :
Intending to search for "Didi Sports Watch"
"didi" -> suggestion "duti" with no matches

Intending to search for "Bottle"
"bot" -> suggestion for "but" and "boi", the latter bringing no search results

Intending to serach for "Summit
"summ" -> suggestion "seam" and "sued" which bring results but not the watch
Manual testing scenarios (*)

    install sample data package
    type in following terms in searchbox on store front and verify it don't suggest autocomplete with strange result:

    "didi" -> does not suggestion "duti"
    "bot" -> does not suggest "boi"
    "summ" -> does notsuggestion "seam" or "sued"
